django_project/utils/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from . import views
+
+"""
+	TODO: 
+	1. Add URL subpaths here.
+	2. Write functions in views.py to handle the various subpaths that you added.
+"""
+
+urlpatterns = [
+    path('', views.base_page, name='basePage'),
+    path('post/<post_id>', views.post_page, name='posts'),
```

Comment:
This is a pretty small thing but I think it might influence people who build off of this starter code. With django path variables, you can have types. For example if you do `post/<int:post_id>`, the post_id variable in the url must be an int, otherwise it won't match the url. It will also do the type conversion so it will pass an `int` to the `post_page` function. This is a good practice and I think putting in the starter code would remind people that types exist and they should use them whenever possible.